configs/coins/bsc_archive.json

```diff
@@ -0,0 +1,73 @@
+{
+  "coin": {
+    "name": "BNB Smart Chain Archive",
+    "shortcut": "BNB",
+    "label": "BNB Smart Chain",
+    "alias": "bsc_archive"
+  },
+  "ports": {
+    "backend_rpc": 8065,
+    "backend_p2p": 38365,
+    "backend_http": 8165,
+    "blockbook_internal": 9065,
+    "blockbook_public": 9165
+  },
+  "ipc": {
+    "rpc_url_template": "ws://127.0.0.1:{{.Ports.BackendRPC}}",
+    "rpc_timeout": 25
+  },
+  "backend": {
+    "package_name": "backend-bsc-archive",
+    "package_revision": "satoshilabs-1",
+    "system_user": "bsc",
+    "version": "1.1.18-hf",
+    "binary_url": "https://github.com/bnb-chain/bsc/releases/download/v1.1.18_hf/geth_linux",
+    "verification_type": "sha256",
+    "verification_source": "1427ef4bc19e51fe9dead2d211ff6ebebe610cbbe7a0e1b35870f01cfed6473e",
+    "extract_command": "mv ${ARCHIVE} backend/geth_linux && chmod +x backend/geth_linux && echo",
+    "exclude_files": [],
+    "exec_command_template": "/bin/sh -c '{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/bsc_exec.sh 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
+    "exec_script": "bsc.sh",
+    "logrotate_files_template": "{{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log",
+    "postinst_script_template": "wget https://github.com/bnb-chain/bsc/releases/download/v1.1.18_hf/mainnet.zip -O {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/mainnet.zip && unzip -o -d {{.Env.BackendInstallPath}}/{{.Coin.Alias}} {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/mainnet.zip && rm -f {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/mainnet.zip",
+    "service_type": "simple",
+    "service_additional_params_template": "",
+    "protect_memory": true,
+    "mainnet": true,
+    "server_config_file": "",
+    "client_config_file": "",
+    "platforms": {
+      "arm64": {
+        "binary_url": "https://github.com/bnb-chain/bsc/releases/download/v1.1.18_hf/geth-linux-arm64",
+        "verification_source": "a7ef543567c012622c9069b3687e413fa86a8465e6b80302803b303c5a71fd9d"
+      }
+    }
+  },
+  "blockbook": {
+    "package_name": "blockbook-bsc-archive",
+    "system_user": "blockbook-bsc",
+    "internal_binding_template": ":{{.Ports.BlockbookInternal}}",
+    "public_binding_template": ":{{.Ports.BlockbookPublic}}",
+    "explorer_url": "",
+    "additional_params": "-workers=16",
+    "block_chain": {
+      "parse": true,
+      "mempool_workers": 8,
+      "mempool_sub_workers": 2,
+      "block_addresses_to_keep": 600,
+      "additional_params": {
+        "address_aliases": true,
+        "mempoolTxTimeoutHours": 48,
+        "processInternalTransactions": true,
+        "queryBackendOnMempoolResync": false,
+        "fiat_rates": "coingecko",
+        "fiat_rates_vs_currencies": "AED,ARS,AUD,BDT,BHD,BMD,BRL,CAD,CHF,CLP,CNY,CZK,DKK,EUR,GBP,HKD,HUF,IDR,ILS,INR,JPY,KRW,KWD,LKR,MMK,MXN,MYR,NGN,NOK,NZD,PHP,PKR,PLN,RUB,SAR,SEK,SGD,THB,TRY,TWD,UAH,USD,VEF,VND,ZAR,BTC,ETH",
```

Comment:
You removed the `fourByteSignatures` entry. Do you think the signatures do not work for BSC?

Reply:
Bad copy, good catch. Although I did notice there looks to be issues handling conflicting sighashes when resolving the name field. Seems difficult outside of best guess without having abi available...
![image](https://user-images.githubusercontent.com/35275952/217025452-2f986210-810c-4047-b3e1-148be5a9738e.png)

Reply:
https://github.com/trezor/blockbook/pull/869/commits/b3da7aa10789a4fc596838a5267fe562938a693d

Reply:
What method should it be? What tx is it?

Reply:
an example tx would be: https://bscscan.com/tx/0xcdc4a1495c6734af04abc1955fedff6837d09aa013e9d7fbb927be28b663cc23

so just a standard `transfer` method. it may just be related to the 4 byte storage as I am looking. I haven't had long term syncs during dev testing, so possible I just haven't stored that sighash yet from the 4bytes directory.

Reply:
It seems plausible that your 4 byte directory is not yet synced as the data look like a standard transfer.

Reply:
For posterity, it finally populated:
![image](https://user-images.githubusercontent.com/35275952/217114215-f8e591fa-dc03-483a-8cf2-36ce3d8114f2.png)